pkgs/tools/networking/corerad/default.nix

```diff
@@ -1,24 +1,28 @@
-{ stdenv, buildGoModule, fetchFromGitHub }:
+{ stdenv, buildGoModule, fetchFromGitHub, nixosTests }:
 
 buildGoModule rec {
   pname = "corerad";
-  version = "0.2.6";
+  version = "0.2.7";
 
   src = fetchFromGitHub {
     owner = "mdlayher";
     repo = "corerad";
     rev = "v${version}";
-    sha256 = "16rwydvqkzi0jlgwpl3d4f8zd35y4lv4h5xa30ybqmwwp1k5ymf0";
+    sha256 = "073hjbwra8ihh11ha5ajqq2r98cc5li4k0xs4p4s055q197zj3aa";
   };
 
-  vendorSha256 = "1431fvi9b0id3zhgkxhiampc5avvp998lncyd5l2gn5py3qz6sdl";
+  vendorSha256 = "19hp8xqr50v8h9vblihalvkb9ll8c0v4p071j9j1zkbjhnb07rca";
 
   buildFlagsArray = ''
     -ldflags=
-    -X github.com/mdlayher/corerad/internal/build.linkTimestamp=1591474872
+    -X github.com/mdlayher/corerad/internal/build.linkTimestamp=1593050100
```

Comment:
I'd suggest removing/disabling this or setting it to `SOURCE_DATE_EPOCH` as if anyone else (human or bot) ever bumps this package I doubt it will be updated.

Reply:
Yep, this is something I've been meaning to automate but I haven't actually figured out how yet.

However, I did just find https://nixos.org/nixpkgs/manual/#chap-stdenv:

> set-source-date-epoch-to-latest.sh
> This sets SOURCE_DATE_EPOCH to the modification time of the most recent file.

So perhaps I can figure out how to plumb this in. Should I do it in this PR?

Reply:
No, doesn't need to be done in this PR.

TBH, I wouldn't bother with trying to set a timestamp unless it's somehow necessary.

Reply:
I've found a method that seems to work. I previously added a file in the root of the git repository which is updated with the current tag's timestamp when I make a release. By reading this file in preBuild, I can automate the process. Let me know if this is alright.

Reply:
Does this work?
```
build.linkTimestamp=$(<.gittagtime)
```

Reply:
It does, thanks! Updated.